crates/fluvio-dataplane-protocol/src/batch.rs

```diff
@@ -185,7 +184,7 @@ impl TryFrom<Batch> for Batch<RawRecords> {
         let compression = f.get_compression()?;
         let compressed_records = compression.compress(&buf)?;
```

Comment:
I think that we can make `compress()` to return `Bytes` instead of `Vec<u8`

Reply:
Great idea. Added it.